docs/repos/git/review-pull-requests.md

```diff
@@ -92,14 +92,19 @@ You can manage PRs in Azure DevOps Services by using the [Azure DevOps command l
 
 ::: moniker range=">= azure-devops-2022"
 
-Pull requests let designated reviewers examine, discuss, and vote on proposed changes before those changes are applied to a target branch of a repo. The following steps describe how reviewers of PRs in Azure Repos can navigate through a PR to understand the proposed changes.
+Pull requests let designated reviewers examine, discuss, and vote on proposed changes before those changes are applied to a target branch of a repo. Keep in mind that if development is carried out with the same files on different branches there could be a conflicts in changes. In that case it is required to carefully analyze and fix those conflicts. The solving can be selectively applied so that all changes are included (not just press on take source for whole file as an example). This can be done using ADO tools, or it will be more convenient to use development tools (e.g. Visual Studio).
```

Comment:
I'd say
"there may be conflicting changes" instead of "there could be a conflicts in changes"
"analyze and resolve those conflicts" instead of "analyze and fix those conflicts"
"To do so, may require manually editing the file to correctly apply the changes from either source, target or both branches. Taking all changes from either source or target (which can be done by simple click on a button) may not be optimal and can result in loss of some of the required changes." instead of "The solving can be selectively applied so that all changes are included (not just press on take source for whole file as an example)."